backend/internal/usecase/transaction.go

```diff
@@ -145,3 +150,41 @@ func (uc *TransactionUsecase) QueryIncludedTransactions(ctx context.Context, txL
 	}
 	return txs, nil
 }
+
+func (uc *TransactionUsecase) QueryTransactionDetail(ctx context.Context, txHash string) (*QueryTransactionDetailResp, *error.Http) {
+	erpcTX, err := uc.erpcDBQuery.QueryFromTransactionDetail(ctx, txHash)
```

Comment:
What happens in the case where the txHash cannot be found in the ERPC database (which is possible)

Reply:
yeah in the call we decided to return not found on the page. Need to adjust this

Reply:
handled the case